src/engine/client/textrender.cpp

```diff
@@ -0,0 +1,1169 @@
+/* (c) Magnus Auvinen. See licence.txt in the root of the distribution for more information. */
+/* If you are missing that file, acquire a complete release at teeworlds.com.                */
+#include <base/system.h>
+#include <base/math.h>
+#include <engine/external/json-parser/json.h>
+#include <engine/graphics.h>
+#include <engine/textrender.h>
+
+#include "textrender.h"
+
+int CAtlas::TrySection(int Index, int Width, int Height)
+{
+	ivec3 Section = m_Sections[Index];
+	int CurX = Section.x;
+	int CurY = Section.y;
+
+	int FitWidth = Width;
+
+	if (CurX + Width > m_Width - 1) return -1;
```

Comment:
while you are at it you could update all these ``if (`` and ``for (`` to ``if(`` and ``for(``

Reply:
Just in time before I made a commit. Changed.